README.md

```diff
@@ -91,15 +91,14 @@ import rebuild from 'electron-rebuild';
 A full build process might look something like:
 
 ```js
-let childProcess = require('child_process');
-let pathToElectron = require('electron-prebuilt');
-
-  rebuild(__dirname, '1.4.12')
-    .then(() => console.info('Rebuild Successful'))
-    .catch((e) => {
-      console.error("Building modules didn't work!");
-      console.error(e);
-    });
+const rebuild = require('electron-prebuilt');
+
+rebuild(__dirname, '1.4.12')
```

Comment:
On the topic of docs, this won't work 😆 

Because use `babel` and `export` this would have to be

```js
rebuild.default(__dirname, '1.4.12');
```

Super annoying 😢